dotCMS/src/main/java/com/dotcms/workflow/form/FireActionForm.java

```diff
@@ -94,7 +100,9 @@ public FireActionForm(final Builder builder) {
         @JsonProperty()
         private String whereToSend;
         @JsonProperty()
-        private String forcePush;
+        private String filterKey;
```

Comment:
Codacy found an issue: [Field filterKey has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5919676)